### ISSUE TITLE
ProgressIndicator accessibility issues

### ISSUE THREAD

**Issue by betrue-final-final** [OfficeDev/office-ui-fabric-react#1614]
Environment: IE11 and JAWS 18
Are you willing to submit a PR to fix? No. I need help
Requested priority: High

1. It only says its status twice. Reporting of progress should be more granular
Expect: Should say at least three strings; start, some point in the middle, and completion, if not more. Could also use a repeated tone that denotes progress, like JAWS setup does.

2. Doesn't announce completion
Expect: Should say when the operation is complete

3. Doesn't say what the progress is for
Expect: Should announce "<operation name>" at start.

**Comment by betrue-final-final** [comment]
Silent in Narrator as well.

**Comment by betrue-final-final** [comment]
Still is an issue.

**Comment by betrue-final-final** [comment]
@lynamemi,
Someone really needs to fix this. Game?

**Comment by lynamemi** [comment]
It's really mysterious to me why this is silent, the code looks ok! I'll have to take more time to investigate.

**Comment by lynamemi** [comment]
This is a WIP

**Comment by betrue-final-final** [comment]
This is not stale. Emily is working through it.

**Comment by lynamemi** [comment]
Leaving this issue open, because it still very much exists, however also referencing this new issue that could someday help resolve this:
https://github.com/OfficeDev/office-ui-fabric-react/issues/5615

**Comment by micahgodbolt** [comment]
blocked, waiting on work done in #5615
